docs/airflow/production-maintenance.md

```diff
@@ -74,8 +74,7 @@ In addition to the tabular view above, here is a diagram representing DAG depend
       gtfs_downloader-->gtfs_loader;
       gtfs_schedule_history-->gtfs_loader;
       gtfs_loader-->gtfs_schedule_history2;
-      payments_loader-->payments_views_staging;
-      payments_views_staging-->payments_views;
+      payments_loader-->payments_views;
```

Comment:
Same comment here -- if there's no enforced dependency, I think we can remove this (just list each DAG but without arrow linking them)